master/getting-started/openstack/installation/index.md

```diff
@@ -15,23 +15,10 @@ You can install {{site.prodname}} via any of the following methods:
 - [RPM-based install for Red Hat Enterprise Linux 7 (RHEL
   7)]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/redhat)
 
-
-- [Canonical's Juju
-  Charms]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/juju)
-
-- [Mirantis
-  Fuel]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/fuel)
-
-- [DevStack]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/devstack)
-
-The [Chef
-installation]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/chef),
-which we used to recommend, is now very old and only supports Icehouse. We now
-recommend using one of the above methods instead.
+- [DevStack]({{site.baseurl}}/{{page.version}}/getting-started/openstack/installation/devstack) (for development purposes only - not recommended for production!)
```

Comment:
Here, you should paste in the actual em dash character and ensure that there are no spaces around it.